rpi-eeprom-update

```diff
@@ -288,7 +288,7 @@ getBootloaderUpdateVersion() {
    match=".*/pieeprom-[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9].bin"
    latest="$(find "${FIRMWARE_IMAGE_DIR}/" -maxdepth 1 -type f -size "${EEPROM_SIZE}c" -regex "${match}" | sort -r | head -n1)"
    if [ -f "${latest}" ]; then
-      BOOTLOADER_UPDATE_VERSION=$(strings "${latest}" | grep BUILD_TIMESTAMP | sed 's/.*=//g')
+      BOOTLOADER_UPDATE_VERSION=$(cat "${latest}" | tr -c [:print:] "\n" | grep BUILD_TIMESTAMP | sed 's/.*=//g')
```

Comment:
What about saving some forks and execs?
* This `cat` is unnecessary. `tr` gets its input from standard input, so simply use shell redirection.
* `sed` can match by pattern, so you need no `grep`.

You may also want to be slightly more careful with the regexp:

```
       BOOTLOADER_UPDATE_VERSION=$(tr -c [:print:] "\n" < "${latest}" | sed '/^BUILD_TIMESTAMP=/s/.*=//p;d')
```

Reply:
Thanks for the comments, I'll test everything and update the patch.